src/app/Dashboard/components/LogPointsModal.js

```diff
@@ -0,0 +1,188 @@
+import TextField from '@material-ui/core/TextField';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import IconButton from '@material-ui/core/IconButton';
+import StartIcon from '@material-ui/icons/Star';
+import EventIcon from '@material-ui/icons/Event';
+import MenuItem from '@material-ui/core/MenuItem';
+
+import actions from '../operations/actions';
+
+/**
+   * @name LogActivityForm
+   * @summary Returns Form
+   * @returns Returns a form
+   */
+class LogActivityForm extends Component {
+    static defaultProps = {
+      categories: [],
+    }
+
+  static propTypes = {
+    show: PropTypes.bool.isRequired,
+    logActivity: PropTypes.func.isRequired,
+    categories: PropTypes.arrayOf(PropTypes.shape({})),
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      categoryOption: [],
+      numberOfParticipants: '',
+      description: '',
+      activityDate: '',
+    };
+  }
+
+  /**
+   * @name requiresNumberOf
+   * @summary determine whether a category should have a number of input
+   * @return {Boolean} whether a category should have a number of input
+   */
+
+  handleChange = prop => (event) => {
+    this.setState({ [prop]: event.target.value });
+  };
+
+  render() {
+    const {
+      categoryOption, numberOfParticipants, description, activityDate,
+    } = this.state;
+    const { show, logActivity, categories } = this.props;
+    console.log('the categories', categoryOption.id);
+    console.log('the number of part', numberOfParticipants);
+    console.log('the desacription', description);
+    console.log('the date', activityDate);
+    console.log('the state', categories);
+    return (
+      <div>
+        <section className='login-jumbotron'>
+          <div
+            className='login-container'
+            style={{
+              transform: show ? 'translateY(0vh)' : 'translateY(-100vh)',
+              opacity: show ? '1' : '0',
+            }}
+          >
+            <div className='log-points-ratings'>
+              <StartIcon className='ratings-icon' />
+              <StartIcon className='ratings-icon' />
+              <StartIcon className='ratings-icon' />
+            </div>
+            <div className='log-points'>
+              <h5>Log in points</h5>
+            </div>
+            <form className='form-container'>
+              <TextField
+                value={categoryOption}
+                select
+                label='Category'
+                margin='normal'
+                fullWidth
+                onChange={this.handleChange('categoryOption')}
+              >
+                {categories.map(option => (
+                  <MenuItem key={option.id} value={option}>
+                    {option.name}
+                  </MenuItem>
+                ))}
+              </TextField>
+              {
+                categoryOption.supportsMultipleParticipants === true
+                  ? (
+                    <Fragment>
```

Comment:
I do not think it's necessary to wrap `Textfield` in a `Fragment`. If I am wrong let me know.

Reply:
You are not wrong @Chris-Maina  there was no need of using Fragment